Wikipedia/Code/WMFFindInPageKeyboardBar.m

```diff
@@ -16,6 +16,18 @@ @interface WMFFindInPageKeyboardBar() <UITextFieldDelegate>
 
 @implementation WMFFindInPageKeyboardBar
 
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    [self hideCopyPasteIcons];
+}
+
+- (void)hideCopyPasteIcons {
```

Comment:
We should probabaly check with Rita on this, but its standard and useful… not sure if we just rip them out.